ui/src/app/components/resource-actions-menu.tsx

```diff
@@ -45,11 +49,7 @@ export const ResourceActionsMenu = (props: {
         )
       }
     >
-      {props.appsAnalysis ? (
```

Comment:
The only thing this prop does is choose the menu component, so I think it would be better to be explicit about this.